src/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -137,7 +147,19 @@ export class SummaryPanelNode extends React.Component<SummaryPanelNodeProps, Sum
   };
 
   // TODO:(see https://github.com/kiali/kiali-design/issues/63) If we want to show an icon for SE uncomment below
-  private renderBadgeSummary = (hasCB?: boolean, hasVS?: boolean, hasMissingSC?: boolean, isDead?: boolean) => {
+  private renderBadgeSummary = (
+    hasCB?: boolean,
+    hasVS?: boolean,
+    hasMissingSC?: boolean,
+    isDead?: boolean,
+    hasRequestRouting?: boolean,
+    hasFaultInjection?: boolean,
+    hasTrafficShifting?: boolean,
+    hasTCPTrafficShifting?: boolean,
+    hasRequestTimeout?: boolean
+  ) => {
```

Comment:
With this many params would be nice if they were alphabetized :-)  No change required, though.

Reply:
I turned the param into an object so it doesn't matter in what order they are passed but that did make the definition kind of gnarly. Let me know what you think.

Reply:
Yeah, I don't love that, sorry for opening a can of worms :-)  How about something similar:

```
  private renderBadgeSummary = (nodeData: DecoratedGraphNodeData) => {
    const {
      hasCB,
      hasFaultInjection,
      hasMissingSC,
      hasRequestRouting,
      hasRequestTimeout,
      hasTCPTrafficShifting,
      hasTrafficShifting,
      hasVS,
      isDead
    } = nodeData;

```

Reply:
That's much better. I'll update.